channels/it.m3u

```diff
@@ -634,6 +640,8 @@ http://wms.shared.streamshow.it/veratv/mp4:veratv/playlist.m3u8
 http://wms.shared.streamshow.it/veratv/veratv/playlist.m3u8
 #EXTINF:-1 tvg-id="VeraTVMarche.it" tvg-name="Vera TV Marche" tvg-country="IT" tvg-language="Italian" tvg-logo="https://i.imgur.com/kXDdx0h.png" group-title="",Vera TV Marche (480p)
 http://wms.shared.streamshow.it:1935/veratv/veratv/playlist.m3u8
+#EXTINF:-1 tvg-id="VH1.it" tvg-name="VH1" tvg-country="IT" tvg-language="" tvg-logo="https://i.imgur.com/5ONlZGS.png" group-title="",VH1 (1080p)
```

Comment:
VH1 is from Viacom Enterprises, American-based channel, even if this version is, indeed, an Italian variant only hosted in Italia.